deploy/smcp-templates/servicemesh

```diff
@@ -31,4 +31,4 @@ spec:
       image: prometheus-rhel8
   threeScale:
     hub: registry.redhat.io/openshift-service-mesh
-    tag: 1.0.0
```

Comment:
Did you mean to remove the tag?

Reply:
I did at the time because I was considering always pulling latest but after some thought I reverted that

Reply:
We wouldn't be using latest in a production environment so should include the explicit tag or a floating tag

Reply:
Yep, I came to that realisation and reverted the change